Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Results/ResultsViewController.swift

```diff
@@ -0,0 +1,60 @@
+import UIKit
+
+/// View controller for `ResultsView` screen.
+public final class ResultsViewController: UIViewController {
```

Comment:
As you highlighted this morning, we need to decide whether to figure out the merge order for this or https://github.com/govuk-one-login/mobile-ios-common/pull/27 

If the other `BaseViewController` PR comes first, then this PR should likely be updated to inherit from `BaseViewController` and the view model protocol would need updating too. Happy to catch up about that if / when it happens if that would be helpful.